docs/meetups/07-18-2023.md

```diff
@@ -17,3 +17,24 @@
    - Sm75.
    - device functions. How hard is this to support while Triton frontend traverses AST?
    - remove torch dependencies from the frontend. (it sounds like there is already progress on this but could be worth discussing)
+
+##### Minutes
+Recording link [here](https://drive.google.com/file/d/1uMlIvih_E5FITwPnNHwTYzo-UKqtey2c/view)
+
+1. Backend plans/broader roadmap:
+   - Plan is for major updates to come in the Triton development meetup which will happen mid-September. For major design changes, currently the plan is to not upstream them directly but have a staging state and different backends can be integrated through a plugin mechanism where Triton provides a layer at the Triton IR layer that is generic and other backends can plug into that.
+   - Short term roadmap plans are very focused on things like helper support, improving all FP8 things on Ampere and Hopper support (end of August). After Hopper support lands, priorities will include refactoring codebase to increase maintainability.
```

Comment:
"on things like helper support", is that a typo for "hopper" ? If yes, seems redundant with the rest of the sentence